pkg/doc/jose/encrypter.go

```diff
@@ -603,3 +888,24 @@ func computeAuthData(protectedHeaders map[string]interface{}, aad []byte) ([]byt
 
 	return output, nil
 }
+
+func jwkMarshalEPK(protectedHeadersJSON map[string]json.RawMessage) error {
```

Comment:
This function is crucial in building the EPK in the ProtectedHeaders when decryption (the same way as encryption)